bin/test_runner.rb

```diff
@@ -58,21 +71,43 @@ def is_up?
   end
 
   def get_line
-    return "#{@stage} #{@name} #{@percepts[:dup]} #{@percepts[:job_status]} #{@percepts[:destroyer_status]}"
+    return "#{@stage} #{@name} #{@percepts[:dup]} #{@percepts[:job_status]} #{@percepts[:destroyer_status]} #{@percepts[:build_id]} #{@percepts[:build]}"
   end
 
   def done?
     @stage == Done || @stage == Failed || @stage == ErrorState
   end
 
-  def update_percepts
+  def init_percepts
     @percepts[:dup]        = is_up?
     @percepts[:job_status] = job_status
     @percepts[:destroyer_status] = destroyer_status
+    @percepts[:build_id]   = job_id
+    @percepts[:build]      = "same"
+  end
 
+  def update_percepts
+    @percepts[:dup]        = is_up?
```

Comment:
nitpic - Understood this was just a matter of copy/paste - but rubocop will complain about this.